### PR TITLE
DW-2120: Support SSM config values

### PR DESCRIPTION
**Pull request by bhomnick**

### CODE REVIEW

File: cdk.context.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (bhomnick) [inline cdk.context.json:1]: @rganeshan-noodle Should this file be included int he repo?  Seems deploy-specific.

**Comment by bhomnick** [comment]
@rganeshan-noodle Some outstanding issues 6/9:

- It doesn't look like the `airflow-database-1` instance is reachable by the tasks anymore.  Perhaps Jeff is still making configuration changes?

File: airflow_stack/airflow_stack.py [hunk 2]
```diff
@@ -96,6 +99,27 @@ def __init__(self, scope: core.Construct, id: str, deploy_env: str, db_redis_sta
...
+            args['NextToken'] = res.get('NextToken')
+            params += res['Parameters']
+            if not args['NextToken']:
+                break
+
+        secrets = {}
+        for param in params:
+            param_name = param['Name'].split('/')[-1]
+            ssm_param = StringParameter.from_secure_string_parameter_attributes(
+                self, f'SSMParam-{param_name}-{self.deploy_env}', version=param['Version'],
```
Comment (bhomnick) [inline airflow_stack/airflow_stack.py:117]: @rganeshan-noodle I don't believe using `valueFrom` for an ECS parameter locks it to a particular SSM version, even though it's specified here for the lookup.  If that's the case we should just be able to restart the service to fetch updated config values.

File: cdk.context.json (hunk 1 above)
Comment (rganeshan-noodle) [reply cdk.context.json:1]: No need to include. I should have added it to .gitignore.

File: airflow_stack/airflow_stack.py (hunk 2 above)
Comment (rganeshan-noodle) [reply airflow_stack/airflow_stack.py:117]: 'cdk deploy <stack>' should also generate the right changeset to restart the containers to pickup config changes. I have not tested this myself. You could try testing after changing the db params to be invalid and see that it fails in cloudwatch logs?

Comment (bhomnick) [reply airflow_stack/airflow_stack.py:117]: It detects a change if we add or remove a parameter, but not if a parameter version changes.  I have a feeling `version` is another vestigial lookup argument that could be any dummy value.  If we update existing config values we'll just need to restart the services, so we could add an invoke task for that.

Comment (rganeshan-noodle) [reply airflow_stack/airflow_stack.py:117]: Yes - you are right. I think there may already be an invoke task to stop tasks.

**Comment by bhomnick** [comment]
@rganeshan-noodle This is ready for review/merging if approved.

**Pull request merged**